config_default.yml

```diff
@@ -12,6 +12,9 @@ disableFlash: true
 jsErrorsEnabled: false
 logEnabled: true
 takeScreenshots: true
+#Specify browser window size in format <width>x<height>, example: 800x600
+#use "max" value if you want browser to be maximised
+browserSize: max
```

Comment:
Consider to change `max` to `maximised`.
`max` could mean to set browser size to maximum value which VM supported.